tsl/test/expected/continuous_aggs_ddl.out

```diff
@@ -186,24 +189,6 @@ SELECT * FROM rename_c_aggregate;
 (0 rows)
 
 ALTER VIEW rename_schema.:"PART_VIEW_NAME" RENAME TO partial_view;
-SELECT user_view_schema, user_view_name, partial_view_schema, partial_view_name,
-      direct_view_schema, direct_view_name
-      FROM _timescaledb_catalog.continuous_agg;
- user_view_schema |   user_view_name   | partial_view_schema | partial_view_name | direct_view_schema | direct_view_name 
-------------------+--------------------+---------------------+-------------------+--------------------+------------------
- public           | rename_c_aggregate | rename_schema       | partial_view      | public             | _direct_view_3
-(1 row)
-
---rename direct view
-ALTER VIEW :"DIR_VIEW_NAME" RENAME TO direct_view;
-SELECT user_view_schema, user_view_name, partial_view_schema, partial_view_name,
-      direct_view_schema, direct_view_name
-      FROM _timescaledb_catalog.continuous_agg;
- user_view_schema |   user_view_name   | partial_view_schema | partial_view_name | direct_view_schema | direct_view_name 
-------------------+--------------------+---------------------+-------------------+--------------------+------------------
- public           | rename_c_aggregate | rename_schema       | partial_view      | public             | direct_view
-(1 row)
-
```

Comment:
I don't follow why this part is affected. Isn't it useful to check the result of the view rename? Does the other test of direct view rename exist already?

Reply:
There is an earlier test that checks that rename of the schema works using `ALTER VIEW`. Not really sure what this is supposed to test, that we do not block `ALTER VIEW`?

Reply:
I have two confusions here:
1. The PR is about increasing coverage, but it also removes tests, which is confusing. Can you provide link in the code where the test exists?
2. The change keeps an alter statement, but removes the select, which checks the result of alter. This looks to me odd.

Reply:
1. No, there is no explicit test of `ALTER VIEW` for the direct view, but we do not have tests for other view operations on the direct view either because it's just a normal SQL statement.
2. Yeah, but since it is just a normal `ALTER VIEW` it will work if it does not emit an error.

Adding it back since you seem to be concerned about it, but I do not find the test useful.